README.md

```diff
@@ -151,6 +151,24 @@ UnSubscribing from channel messages
 await channelMessageSubscription.cancel();
 ```
 
+Publishing channel messages
+
+```dart
+//publish name and data
+await channel.publish(name: "event1", data: "hello world");
+await channel.publish(name: "event1", data: {"hello": "world", "hey": "ably"});
+await channel.publish(name: "event1", data: [{"hello": {"great": "world"}, "ably": {"serious": true, "realtime": true}, "key3": null}, "string4", "string5"]);
```

Comment:
This line should be shortened. It requires horizontal scrolling when rendered
https://github.com/ably/ably-flutter/blob/994979557e4288f42412a0ddcc2c714077a01291/README.md#realtime-api

Reply:
addresseed f0e1f306a2e72900b44d5278a97244300c9372e4